api/src/main/kotlin/io/spine/protodata/renderer/Renderer.kt

```diff
@@ -74,4 +75,21 @@ protected constructor(
     final override fun <T> configAs(cls: Class<T>): T = super.configAs(cls)
 
     final override fun configIsPresent(): Boolean = super.configIsPresent()
+
+    /**
+     * Injects the context of the ProtoData application.
+     *
+     * This method is `public` but is essentially `internal` to ProtoData SDK.
+     * It is annotated `@Internal` to prevent its accidental use by the users of the SDK.
+     *
+     * This method must be called by ProtoData only once.
+     * Repeat calls will result in `IllegalStateException`.
+     */
+    @Internal
+    public fun injectContext(protoDataContext: BoundedContext) {
+        check(!this::protoDataContext.isInitialized) {
```

Comment:
Please use the `ContextAware` interface along with the check methods provided there.